modules/ROOT/pages/proxy-latest-concept.adoc

```diff
@@ -1,30 +1,66 @@
 
-* RAML/OAS Proxy
-* WSDL Proxy
-* HTTP Proxy
-* HTTPS Proxy
+== Prerequisites
 
-API Manager automatically generates the proxy app when you configure the API as an endpoint with a proxy and includes an Autodiscovery element in the app. Mule locks the API until all policies have been applied. The client app (consumer) calls the proxy which forwards the call to the API. After you deploy the app, Mule Runtime calls API Manager using the client ID and secret to get the policies for the API.
+Before you deploy your API proxy, ensure that you complete the following requirements based on where you are deploying the proxy:
 
-In most cases, the proxy you generate in API Manager is suitable for deployment. However, you can modify the proxy to log data to a file or send data to a Splunk account with the Anypoint Splunk Connector, for example.
+* Hybrid: 
+** For private servers, you must have access to a Mule runtime engine (mule) instance that has been configured for your organization. 
++
+For information on how to configure a Mule instance as a server, see xref:runtime-manager::servers-create.adoc[register a server in Runtime Manager].
+** You must register your private server with a Mule agent that is compatible with your Mule instance.
++
+For Mule versions 3.8 and later, use Mule Agent 1.9.0. For Mule 4.x, use the Mule Agent specified in the release notes. 
+* Runtime Fabric
+** You must have the Manage APIs Configuration and Deploy API Proxies permissions on your account.
+** You must be using Mule versions 3.8.7, 3.9.1, or 4.1.2, or later.
+* CloudHub:
++
+Currently, no prerequisites are required for CloudHub
 
-You can import a proxy that you download from API Manager into Studio as a Mule Deployable Archive. You can then modify the proxy in Studio.
+//Kim, IMO, not mentioning that no prerequisites are required for CH might be confusing for the users, so I addedd it. What do you think?
 
-If you configured an endpoint with a proxy based on Mule 3 to use the HTTPS scheme, you need to modify a proxy to add HTTPS credentials following the xref:mule-runtime::build-an-https-service.adoc[Build an HTTPS Service] instructions. +
-Additionally, if you configured a proxy based on Mule 4, you can build an HTTPS proxy using a TLS context stored in Anypoint Security's secrets manager following the xref:building-https-proxy.adoc[Building an HTTPS API Proxy] instructions.
+== Deploy your API Proxy
 
+To deploy your proxy:
 
+. From Anypoint Platform, select *API Manager* > *Manage API* > *Manage API From Exchange*.
+. In *API Name*, search for the name of the asset as saved in Exchange.
++
+The *Asset type*, *API version*, and *Asset version* fields for the API are auto-populated from Exchange.
```

Comment:
Although this is interesting, is it necessary to the task?

Reply:
I'd leave it in. Done! Thanks!